create-custom-map-style/example-style-descriptor.json

```
@@ -0,0 +1,7381 @@
+{
+  "version": 8,
+  "sources": {
+    "esri": {
+      "attribution": "Esri, HERE, Garmin, FAO, NOAA, USGS, © OpenStreetMap contributors, and the GIS User Community",
+      "maxzoom": 15,
+      "tiles": ["http://localhost:8080/{z}/{x}/{y}@2x.png"],
```

After https://github.com/beatleboy501/tilelive-aws/pull/1, this can eliminate some weirdness:

```suggestion
      "tiles": ["http://localhost:8080/{z}/{x}/{y}.pbf"],
```